io/jvm/src/main/scala/org/scalajs/io/AtomicFileOutputStream.scala

```diff
@@ -40,14 +40,13 @@ private[io] class AtomicFileOutputStream private (
 
   /** Try to atomically replace the baseFile with the tmpFile */
   private[this] def atomicReplace(): Unit = {
-    if (!tmpFile.renameTo(baseFile)) {
-      // Renaming failed. Fallback to copy
-      try {
-        IO.copyTo(FileVirtualBinaryFile(tmpFile),
-            WritableFileVirtualBinaryFile(baseFile))
-      } finally {
-        tmpFile.delete()
-      }
+    try {
+      // Try atomic move.
+      Files.move(tmpFile.toPath, baseFile.toPath, StandardCopyOption.ATOMIC_MOVE)
+    } catch {
+      case _: AtomicMoveNotSupportedException =>
+        // Fallback to normal copy.
+        Files.copy(tmpFile.toPath, baseFile.toPath, StandardCopyOption.REPLACE_EXISTING)
```

Comment:
I think that in the fallback, we should still `tmpFile.delete()` after the `Files.copy`.

Reply:
Also, there's a typo in the commit message: Automic